client/post-editor/editor-drawer/index.jsx

```diff
@@ -251,6 +253,11 @@ class EditorDrawer extends Component {
 			if ( ! this.props.isSeoToolsModuleActive || ! jetpackVersionSupportsSeo ) {
 				return;
 			}
+
+			// Don't show SEO accordion if this setting is managed by other SEO plugin.
+			if ( !! getConflictingSeoPlugins( activePlugins ).length ) {
```

Comment:
We can probably move this down to `mapStateToProps` so we don't need to do this on each render.

Reply:
Moved in dd40d7aea39fb7a2ab6bf69f101c97240ddc8cc1.